Sources/gg.datagram.web-requests.sdPlugin/app.js

```diff
@@ -98,8 +98,13 @@ function sendWebSocket(data) {
     ws.onopen = function() {
         onOpen(this);
         ws.send(body);
-        ws.close();
         showOk(data.context);
```

Comment:
It would be nice to only show the success message once we've confirmed that the data has been sent, and show an alert if there's still buffered data after maybe 3s.